src/sentry/static/sentry/app/views/alerts/details/index.tsx

```diff
@@ -152,8 +153,11 @@ class IncidentDetails extends React.Component<Props, State> {
           onSubscriptionChange={this.handleSubscriptionChange}
           onStatusChange={this.handleStatusChange}
         />
-
-        <DetailsBody {...this.props} incident={incident} stats={stats} />
+        {organization.features.includes('alert-details-redesign') ? (
```

Comment:
Not sure if the standard is to use the `<Feature>` component here, but this seems fine since it'll be a temporary measure while we build the redesign out.